test/ProfileTracerTest.cpp

```diff
@@ -44,6 +46,9 @@ void ProfileTracerTest::SetUp(void)
         m_data.push_back({time, rank, event, region_hash});
         time += 1.0;
     }
+
+     m_data.push_back({40, 0, geopm::EVENT_SHORT_REGION, 0xdeadbeef});
```

Comment:
```suggestion
     m_data.push_back({40, 0, geopm::EVENT_SHORT_REGION, 88});
```
in general the event number will not match the region hash